search-app/resources/bin_community_usage.groovy

```diff
@@ -0,0 +1,7 @@
+/* Elastic groovy script to round the community usage to the nearest binSize.
+   Round to 2 decimal places to avoid slight differences from math, which would defeat
+   the purpose of the script.
+*/
+
+def binnedScore = binSize * (double)Math.round(doc['usage-relevancy-score'].value / binSize);
```

Comment:
Dude, you had me there for a millisecond ... slow-motion table-flip stopped mid-flip when I saw "groovy" ;-)